content/en/docs/components/model-registry/getting-started.md

```diff
@@ -163,6 +163,25 @@ isvc = kserve.V1beta1InferenceService(
 ks_client = kserve.KServeClient()
 ks_client.create(isvc)
 ```
+Alternatively, you can create the same InferenceService using a YAML manifest:
+
+```yaml
+apiVersion: serving.kserve.io/v1beta1
+kind: InferenceService
+metadata:
+  name: iris-model
+  namespace: kubeflow  # Replace if different from kserve.utils.get_default_target_namespace()
```

Comment:
```suggestion
  namespace: kubeflow-user-example-com  # Replace if different from kserve.utils.get_default_target_namespace()
```

I'd replace this with the one for the user workload where typically everything is run/created based on standard installation example